Documentation/ApiOverview/Icon/Index.rst

```diff
@@ -40,6 +40,12 @@ The file needs to return a PHP configuration array with the following keys:
 
 ..  index:: Icon API; IconProviderInterface
 
+..  attention::
+    Do not overwrite the $icons variable in this file. This can lead to the error: 
```

Comment:
Please hint me to the $icons variable in the example above, I don't see it. Additionally, please set the attention before the index directive.

Reply:
Never mind, just looked at the code, not at the comments.

Reply:
Ill put the .. attention::  block above the code ^^